internal/tracker/tracker_test.go

```diff
@@ -306,6 +314,81 @@ required-tasks:
       - buildah
       - git-clone
       - summary
+`,
+		},
+		{
+			name: "prefer older entries with same digest",
+			urls: []string{
+				"registry.com/one:1.0@" + sampleHashOne.String(),
+			},
+			input: `---
```

Comment:
As a follow up, I'd like to apply heredoc to these. Skipped for this PR to minimize changes.